package.json

```diff
@@ -23,8 +23,11 @@
     "@nestjs/common": "^10.0.0",
     "@nestjs/config": "^3.2.0",
     "@nestjs/core": "^10.0.0",
+    "@nestjs/mapped-types": "*",
```

Comment:
Using a wildcard (*) for dependency versions in `package.json` can lead to unexpected breaks or changes in behavior as new versions of the dependency are released. It'd be better to specify a fixed version or use a version range that will only capture non-breaking updates